index.html

```diff
@@ -6,15 +6,66 @@
     <title>Quiz App</title>
     <link rel="stylesheet" href="styles/index.css" />
   </head>
-  <body class="app-grid marginzero">
+  <body class="app-grid">
     <header class="app-grid__header">
       <h1>Quiz App</h1>
     </header>
 
-    <main class="app-grid__main"></main>
-    <section class="main__card">
-      <h2>Question</h2>
-    </section>
+    <main class="app-grid__main">
+      <section class="main__card">
+        <section class="card__question-section">
```

Comment:
You could consider using a `header`-element here as it is the header of your card. 
```suggestion
        <header class="card__header">
```